src/hooks/useActiveElementRole/index.js

```diff
@@ -1,20 +1,20 @@
-import {useEffect, useState} from 'react';
+import {useEffect, useRef} from 'react';
 
 /**
  * Listens for the focusin and focusout events and sets the DOM activeElement to the state.
  * On native, we just return null.
  *
- * @return {Element} the active element in the DOM
+ * @return {string | null} the active element in the DOM
  */
-export default function useActiveElement() {
-    const [active, setActive] = useState(document.activeElement);
+export default function useActiveElementRole() {
+    const activeRoleRef = useRef(document.activeElement.role);
```

Comment:
I have used ref here deliberately, since the role prop is being used to pass to the config in `Button` and used in `BaseSelectionList` for disabling enter shortcut.

I have tested it and it looks okay to me, but I will still want to test this thoroughly, so if the reviewer can help me state out the steps on how to test this thoroughly. 👍